fog/view/server/src/router_request_handler.rs

```diff
@@ -168,7 +183,11 @@ where
 
     let mut query_responses: Vec<MultiViewStoreQueryResponse> = Vec::with_capacity(shards.len());
     let mut remaining_tries = RETRY_COUNT;
+    let _timer = BULK_QUERY_REQUESTS.start_timer();
     while remaining_tries > 0 {
+        if remaining_tries < RETRY_COUNT {
+            QUERY_GROUP_RETRY.inc();
```

Comment:
Similar the comment above I think this will be easier to understand below the break statement/if block